examples/internal/proto/examplepb/enum_with_single_value.swagger.json

```diff
@@ -0,0 +1,97 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "examples/internal/proto/examplepb/enum_with_single_value.proto",
+    "version": "version not set"
+  },
+  "tags": [
+    {
+      "name": "EnumWithSingleValueService"
+    }
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/v1/example/foo": {
+      "post": {
+        "operationId": "EnumWithSingleValueService_Echo",
+        "responses": {
+          "200": {
+            "description": "A successful response.",
+            "schema": {
+              "$ref": "#/definitions/examplepbEnumWithSingleValueEchoResponse"
+            }
+          },
+          "default": {
+            "description": "An unexpected error response.",
+            "schema": {
+              "$ref": "#/definitions/rpcStatus"
+            }
+          }
+        },
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/examplepbEnumWithSingleValueEchoRequest"
+            }
+          }
+        ],
+        "tags": [
+          "EnumWithSingleValueService"
+        ]
+      }
+    }
+  },
+  "definitions": {
+    "examplepbEnumWithSingleValue": {
```

Comment:
With this change, we can see that the `enum` field is now omitted from the Swagger spec for this enum that has just a single value.